GPU/GPUTracking/TPCClusterFinder/ClusterAccumulator.cxx

```diff
@@ -13,67 +13,62 @@
 
 #include "ClusterAccumulator.h"
 #include "CfUtils.h"
-#include "ClusterNative.h"
```

Comment:
This change (and some other) triggered some warnings during compilation. See below. Could you fix these:

```
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/GPUTPCClusterFinder.h:26:1: warning: class 'ClusterNative' was previously declared as a struct; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
class ClusterNative;
^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/../../../DataFormats/Detectors/TPC/include/DataFormatsTPC/ClusterNative.h:50:8: note: previous use is here
struct ClusterNative {
       ^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/GPUTPCClusterFinder.h:26:1: note: did you mean struct here?
class ClusterNative;
^~~~~
struct                                                                                                                                                                                                                                                                                                                                                     
In file included from /home/qon/alice/O2/GPU/GPUTracking/Base/hip/GPUReconstructionHIP.hip.cxx:62:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/Base/GPUReconstructionIncludesDevice.h:69:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/GPUTPCClusterFinderKernels.cxx:19:
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Array2D.h:79:1: warning: 'GridSize' defined as a struct template here but previously declared as a class template; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
struct GridSize<1> {
^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Array2D.h:76:1: note: did you mean struct here?
class GridSize;
^~~~~
struct                                                                                                                                                                                                                                                                                                                                                     
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Array2D.h:87:1: warning: 'GridSize' defined as a struct template here but previously declared as a class template; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
struct GridSize<2> {
^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Array2D.h:76:1: note: did you mean struct here?
class GridSize;
^~~~~
struct                                                                                                                                                                                                                                                                                                                                                     
In file included from /home/qon/alice/O2/GPU/GPUTracking/Base/hip/GPUReconstructionHIP.hip.cxx:62:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/Base/GPUReconstructionIncludesDevice.h:69:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/GPUTPCClusterFinderKernels.cxx:25:
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/ClusterAccumulator.h:25:1: warning: class 'ClusterNative' was previously declared as a struct; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
class ClusterNative;
^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/../../../DataFormats/Detectors/TPC/include/DataFormatsTPC/ClusterNative.h:50:8: note: previous use is here
struct ClusterNative {
       ^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/ClusterAccumulator.h:25:1: note: did you mean struct here?
class ClusterNative;
^~~~~
struct                                                                                                                                                                                                                                                                                                                                                     
In file included from /home/qon/alice/O2/GPU/GPUTracking/Base/hip/GPUReconstructionHIP.hip.cxx:62:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/Base/GPUReconstructionIncludesDevice.h:69:
In file included from /home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/GPUTPCClusterFinderKernels.cxx:26:
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Clusterizer.h:27:1: warning: class 'ClusterNative' was previously declared as a class; this is valid, but may result in linker errors under the Microsoft C++ ABI [-Wmismatched-tags]
class ClusterNative;
^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/../../../DataFormats/Detectors/TPC/include/DataFormatsTPC/ClusterNative.h:50:8: note: previous use is here
struct ClusterNative {
       ^
/home/qon/alice/O2/GPU/GPUTracking/Standalone/TPCClusterFinder/Clusterizer.h:27:1: note: did you mean struct here?
class ClusterNative;
^~~~~
struct                                                                            
```